frontend/src/components/Rating/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './rating.scss';
+
+const Rating = props => {
+  const { rating } = props;
+  const roundedRating = Math.round(props.rating);
+  return (
+    <span className="rating" title={rating}>
+      {Array.apply(null, { length: 3 })
+        .map(Number.call, Number)
```

Comment:
That's a really long way of saying `[0, 1, 2]`...

Reply:
I agree, tried to be over dynamic, kinda :)